src/containers/audio-rewards-page/RewardsTile.tsx

```diff
@@ -80,54 +79,46 @@ const useRewardIds = () => {
   const filteredRewards: RewardID[] = rewards.filter(isValidRewardId)
   return filteredRewards
 }
+const rewardsMap = {
```

Comment:
Maybe use a messages variable for the texts here: title, description, etc..

Reply:
yea agree in general. want to keep this PR limited in scope since I'm just ripping out the flag